Apps/Sandcastle/gallery/Particle System.html

```diff
@@ -137,298 +137,301 @@
 </div>
 
 <script id="cesium_sandcastle_script">
-function startup(Cesium) {
-    'use strict';
+    function startup(Cesium) {
```

Comment:
Your changes from you other PR are still in this one, make sure they are removed.